test/e2e/storage/flexvolume.go

```diff
@@ -182,7 +182,7 @@ var _ = utils.SIGDescribe("Flexvolumes", func() {
 		suffix = ns.Name
 	})
 
-	ginkgo.It("should be mountable when non-attachable", func() {
+	ginkgo.It("should be mountable when non-attachable [Feature:Flexvolumes]", func() {
```

Comment:
I know this is for storage feature, but still feel inappropriate to directly add this label.

Reply:
This test is failing constantly, is breaking this job.

It seems a leftover from https://github.com/kubernetes/kubernetes/pull/91329, the linked issue says that these tests should have the feature tag to avoid breaking the e2e suite
https://github.com/kubernetes/kubernetes/issues/78737#issuecomment-627021732

I've added it as a separate commit so the git history is clean and can be reverted independently.

Reply:
cc @msau42 is this okay to do here?

Reply:
The switch the distroless should not have impacted the "non-attachable" test case. This test case is passing fine in the normal k8s job: https://k8s-testgrid.appspot.com/sig-storage-kubernetes#gce&include-filter-by-regex=Flexvolume

So it would be interesting to figure out why it is consistently failing in this job.

Reply:
sorry about that, my mistake